public/app/features/dashboard/components/ShareModal/SharePublicDashboard/ConfigPublicDashboard/EmailSharingConfiguration.tsx

```diff
@@ -139,6 +144,8 @@ export const EmailSharingConfiguration = () => {
   };
 
   const onSubmit = async (data: EmailSharingConfigurationForm) => {
+    //TODO: add if it's domain or not when developed.
```

Comment:
What does it mean?

Reply:
in the next iteration of this feature, we will be managing not only emails but also domains in the input, so a user is able to add a company domain, instead of adding one by one, let's say